app/components/filters_component/filters_component.js

```diff
@@ -38,6 +38,8 @@ export const init = (removeButtonSelector, clearButtonId, closeButtonId, showFil
 
   if (document.documentElement.clientWidth <= MOBILE_BREAKPOINT) {
     setFiltersHiddenState(document.getElementById(showFilterPanelId), document.getElementsByClassName('filters-component')[0], false);
+  } else {
+    filtersUnfocusble();
```

Comment:
Unfocusble? 😉 

(I know that's not new)

Reply:
good spot, will fix as part of this PR